src/ripple/app/ledger/LedgerProposal.h

```diff
@@ -66,23 +64,17 @@ class LedgerProposal
         uint256 const& prevLedger,
         uint256 const& position,
         NetClock::time_point closeTime,
-        NetClock::time_point now);
+        NetClock::time_point now,
+        NodeID const& nodeID);
 
     uint256 getSigningHash () const;
     bool checkSign () const;
 
-    NodeID const& getPeerID () const
-    {
-        return mPeerID;
-    }
-    uint256 const& getCurrentHash () const
-    {
-        return mCurrentHash;
-    }
-    uint256 const& getPrevLedger () const
+    Blob const& getSignature () const
```

Comment:
Please consider using `Buffer` for the member variable, and `Slice` for the return type.